README.md

```diff
@@ -58,3 +58,22 @@ command:
 ```
 firefox com.ibm.streamsx.metrics/doc/spldoc/html/index.html &
 ```
+
+### Building Project
+The MetricsSink operator requires you to build jars and include them in your project. You will also need to have an ElasticSearch server running.
```

Comment:
Why is it required that the user has to build jars, why can't a released toolkit already contain the jars?